File: cs50_x/week6/me/mario.c
// Simple program that makes ascii art of bricks.

#include <stdio.h>

int main(void)
{
    // Gets input from standard input.
    int x, y;
    printf("x: ");
    scanf("%d", &x);
    printf("y: ");
    scanf("%d", &y);

    // Prints output to standard output.
    for (int i = 0; i < y; i++)
    {
        for (int j = 0; j < x; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: cs50_x/week6/me/mario.py
# Simple program that makes ascii art of bricks.

def main():
    # Gets input from standard input.
    x = int(input("x: "))
    y = int(input("y: "))

    # Prints output to standard output.
    for i in range(y):
        print("#" * x)

main() 